azure-kusto-ingest/tests/test_e2e_ingest.py

```diff
@@ -529,15 +529,20 @@ async def test_streaming_ingest_from_dataframe(self):
             "xtextWithNulls",
             "xdynamicWithNulls",
```

Comment:
nit: but column name is not correct anymore (not sure if the table is created ad-hoc or not)